src/Interfaces.ts

```diff
@@ -15,10 +16,17 @@ export interface IResolverValidationOptions {
     requireResolversForAllFields?: boolean;
 }
 
+export interface IResolverOptions {
+    resolve?: GraphQLFieldResolveFn;
+    __resolveType?: IResolveTypeFn;
+    __isTypeOf?: IIsTypeOfFn;
```

Comment:
I think these are two different kinds of deprecation we're talking about. I was talking about graphql field/type deprecation, which was added by Lee Byron with his schema directives.